cmd/webhook/main.go

```diff
@@ -82,6 +83,23 @@ func main() {
 		namespace = "kube-system"
 	}
 
+	if *healthCheckPort < 1024 || *healthCheckPort > 65535 {
```

Comment:
nit: this and the check at L66 are the same, mind putting it into a helper func e.g `isPortValid(port) bool`